### ISSUE TITLE
Bitcoin Design Guide Jam Session #57

### ISSUE THREAD

**Issue by GBKS** [BitcoinDesign/Meta#588]
```meta
UTCTime: 2023-11-06 12:00 UTC
Duration: 1h
```

![bitcoin-design-guide-jam-session-57](https://github.com/BitcoinDesign/Meta/assets/695901/a00adb8f-21f2-47f8-92c7-b69572ad2b46)

### Agenda 

An open session for anyone interested or working on the design guide to chat and collaborate. Leave a comment if you'd like to talk about something specific. Typically, we review the [roadmap](https://github.com/orgs/BitcoinDesign/projects/2), discuss what we're working on, and what we'd like to see happen next. Recently, we also started discussing work on the Bitcoin UI Kit and Bitcoin Icons during this call.

### Check your timezone

https://everytimezone.com/s/88d09b21

### Join the call

https://meet.jit.si/bitcoindesign

### Calendar

Subscribe to the [Bitcoin Design calendar](https://bitcoin.design/calendar/).

**Comment by GBKS** [comment]
A topic I'd like to discuss is the short- to mid-term plans for new additions and changes to the guide. I'll have a document ready we can use as a starting point.

**Comment by GBKS** [comment]
I'd also like to go over the [multi-wallet reference design PR](https://github.com/BitcoinDesign/Guide/pull/1040).

**Comment by rabbitholiness** [comment]
I'd like a quick feedback session about reference designs for Miniscript, a.k.a. complex spending conditions.

**Comment by GBKS** [comment]
Thanks to all who joined. We did not get to all the topics, but covered quite a bit. We discussed:

- Wallet Scrutiny case study
- The new code resources page
- The PR for the new multi-wallet reference design
- Initial designs for a page about recovery path UX
- Descriptors and how to design backup solutions for them

Recordings are up on [BitcoinTV](https://bitcointv.com/w/9Vfa1p3XtS5P7oV6GdifJr) and [YouTube](https://youtu.be/0HhYjG4wYkQ). Notes are [here](https://docs.google.com/document/d/14yUAnfyvnI5PP2Y8rYFie78FIpfp8xUo4vSZhwrkBGg/edit?usp=sharing).
